src/test/java/edu/cornell/kfs/fp/batch/xml/fixture/CuDisbursementVoucherDocumentFixture.java

```diff
@@ -1,18 +1,26 @@
 package edu.cornell.kfs.fp.batch.xml.fixture;
 
+import java.sql.Date;
 import java.util.ArrayList;
+import java.util.Calendar;
 import java.util.List;
 
 import org.apache.commons.lang3.StringUtils;
 import org.kuali.rice.core.api.util.type.KualiDecimal;
 
 import edu.cornell.kfs.sys.fixture.XmlDocumentFixtureUtils;
+import edu.cornell.kfs.sys.xmladapters.StringToJavaDateAdapter;
 
 public enum CuDisbursementVoucherDocumentFixture {
     EMPTY(),
-    JANE_DOE_DV_DETAIL("DISB", "Doe, Jane", "X", "E", 50, "Freeville", "Jane Doe", 25,
+    JANE_DOE_DV_DETAIL("DISB", "Doe, Jane", "X", "E", 50, "Freeville", "Jane Doe", 25, "7/24/2018", StringUtils.EMPTY, StringUtils.EMPTY,
             buildExpenseFixtureArray(CuDisbursementVoucherDocumentNonEmployeeTravelExpenseFixture.DELTA,
                     CuDisbursementVoucherDocumentNonEmployeeTravelExpenseFixture.OTHER_LODGING),
+            buildExpenseFixtureArray(CuDisbursementVoucherDocumentNonEmployeeTravelExpenseFixture.PREPAID_AVIS,
+                    CuDisbursementVoucherDocumentNonEmployeeTravelExpenseFixture.PREPAID_OTHER)),
+    JOHN_DOE_DV_DETAIL("DISB", "Doe, John", "X", "E", 50, "Freeville", "Jane Doe", 25, StringUtils.EMPTY, "04/15/2020", "12321",
```

Comment:
Should the second of the two name strings be "John Doe" instead?

Reply:
yes it should, I shall change